api/tests/integration/application/organizations-administration/organization-administration-controller_test.js

```diff
@@ -1,89 +1,205 @@
+      };
+
+      const response = await organizationAdministrationController.updateOrganizationInformation(request, hFake);
+
+      // then
+      const savedOrganization = await knex('organizations').where('id', organization.id).first();
+
+      expect(response.source.data.type).to.equal('organizations');
+      expect(response.source.data.id).to.equal(organization.id.toString());
+
+      expect(response.source.data.attributes['name']).to.equal(savedOrganization.name);
+      expect(response.source.data.attributes['type']).to.equal(savedOrganization.type);
+      expect(response.source.data.attributes['logo-url']).to.equal(savedOrganization.logoUrl);
+      expect(response.source.data.attributes['email']).to.equal(savedOrganization.email);
+      expect(response.source.data.attributes['credit']).to.equal(savedOrganization.credit);
+      expect(response.source.data.attributes['external-id']).to.equal(savedOrganization.externalId);
+      expect(response.source.data.attributes['province-code']).to.equal(savedOrganization.provinceCode);
+      expect(response.source.data.attributes['is-managing-students']).to.equal(savedOrganization.isManagingStudents);
+      expect(response.source.data.attributes['documentation-url']).to.equal(savedOrganization.documentationUrl);
+      expect(response.source.data.attributes['show-skills']).to.equal(savedOrganization.showSkills);
+      expect(response.source.data.attributes['identity-provider-for-campaigns']).to.equal(
+        savedOrganization.identityProviderForCampaigns
+      );
+    });
+  });
+
+  it('return updated tag organization', async function () {
+    // given
+    const oldTag = databaseBuilder.factory.buildTag({ name: 'my old tag' });
+    const newTag = databaseBuilder.factory.buildTag({ name: 'my new tag' });
+
+    databaseBuilder.factory.buildOrganizationTag({ organizationId: organization.id, tagId: oldTag.id });
+    await databaseBuilder.commit();
+
+    // when
+    const request = {
+      payload: {
+        data: {
+          id: organization.id,
+          attributes: {},
+          relationships: {
+            tags: {
+              data: [{ type: 'tags', id: newTag.id }],
+            },
+          },
+        },
+      },
+    };
+
+    const response = await organizationAdministrationController.updateOrganizationInformation(request, hFake);
 
-    sandbox.stub(securityPreHandlers, 'adminMemberHasAtLeastOneAccessOf');
+    //then
+    const organizationTag = await knex('organization-tags').where('organizationId', organization.id);
 
-    httpTestServer = new HttpTestServer();
-    await httpTestServer.register(moduleUnderTest);
+    expect(organizationTag.length).to.equal(1);
+    expect(response.source.data.relationships.tags.data.length).to.equal(1);
+    expect(response.source.data.relationships.tags.data[0].id).to.equal(newTag.id.toString());
   });
 
-  afterEach(function () {
-    sandbox.restore();
+  it('return updated data officer organization', async function () {
```

Comment:
**Suggestion :**
`it('return updated data protection officer organization', async function () {`